index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <!-- <script defer src="script.js"></script> -->
+    <title>Document</title>
+</head>
+<body>
+
+    <div id="main">
+         <h1>
+            <font color="#0F52BA">M</font>
+            <font color="#FF2626">i</font>
+            <font color="#FFD523">t</font>
+            <font color="#0F52BA">n</font>
+            <font color="#66CC66">i</font>
+            <font color="#FF2626">c</font>
+            <font color="#0F52BA">k</font>
+        </h1>
+
+         <div>
+            <form id="search">
+                <input type="text" id="inputtxt" placeholder="Search..." name="searchBar" size="50">
+                <button type="submit">Search</button> 
+            </form>
+            <form> 
+                <button type="submit" id="app">API</button>    
+            </form>
+        </div>
+        <ul id="results"></ul>
+        <h4 id="fact"></h4>
+    </div>
+    <footer>
+        <p>United Kingdom</p>
+    </footer>
+    <script src="script.js"></script>
```

Comment:
Better to put line 39 in the head, as <script defer src="script.js"></script>, its neater that way.